docker-compose.yml

```diff
@@ -48,9 +48,9 @@ services:
       GIT_BRANCH: ${CI_COMMIT_REF_SLUG}
     volumes:
       - '.:/var/www/html'
-      - ${DOCKER_SHARED_PATH}/composer-cache:/usr/local/composer/cache
-      - ${DOCKER_SHARED_PATH}/logs:/var/log/php
-      - ${DOCKER_SHARED_PATH}/logs:/var/log/silverstripe
+      - ${DOCKER_SHARED_PATH}/.composer-cache:/usr/local/composer/cache
```

Comment:
Any particular reason to change this to a hidden dir ?

Reply:
Outlined in commit:
https://github.com/symbiote/docker-project/pull/8/commits/c30690c52c55bc260645a94b3bbc7ccda8ad30b8

Ultimately I'd just prefer it, but it's your call,

Reply:
The default composer setting is for it to not be hidden, so let's leave it as a public dir.

Reply:
Done.